### ISSUE TITLE
Compliance\ Payment notification sent to CSNR once submitted

### ISSUE THREAD

**Issue by NicoleGovvy** [bcgov/cas-compliance#69]
#### Description:

Once an operator has submitted a payment notification, CSNR finance staff receive an email with the details on when to expect payment, and another email two business days before the payment is expected.

#### Acceptance Criteria:
As a CSNR staff member, when an operator submits a payment notification, given that I receive an email, then I know when to expect payment.

Once the payment notification is submitted in COAM, an email triggers:

Email 1 triggers when the payment notification is submitted: 
- [ ] Email sent to: revenuenrsector@gov.bc.ca
- [ ] Subject line: Payment notification for [Obligation ID]
- [ ] Contains: 
Client Name eLicensing = Operation BCIERS 
Invoice # eLicensing = Compliance ID BCIERS 
Amount 
Date of deposit 

Email 2 triggers two business days before the date of deposit and contains the same information.
- [ ] Email sent to: revenuenrsector@gov.bc.ca
- [ ] Subject line: Upcoming payment notification for [Obligation ID]
- [ ] Contains: 
Client Name eLicensing = Operation BCIERS 
Invoice # eLicensing = Compliance ID BCIERS 
Amount 
Date of deposit 

#### Development Checklist:

- [ ] Checklist item
- [ ] Checklist item
- [ ] Checklist item
- [ ] Meets the DOD

#### Definition of Ready (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

#### Definition of Done (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Blocked By/Blocking
Blocked by #21 

#### Notes:

-

**Comment by vesselak** [comment]
As per BA decisions on January 08, 2025, the payment notification workflow won't be required and email notification requirement would be added to payment instructions for the external user. Closing ticket.
